VTEX - Checkout API.json

```diff
@@ -9039,11 +9039,118 @@
                         "headers": {},
                         "content": {
                             "application/json": {
-                                "schema": {},
+                                "schema": {
+                                    "type": "object",
+                                    "properties": {
+                                        "paymentConfiguration": {
+                                            "type": "object",
+                                            "description": "Payment configuration information.",
+                                            "properties": {
+                                                "requiresAuthenticationForPreAuthorizedPaymentOption": {
+                                                    "type": "boolean",
+                                                    "description": "Determines whether pre-authorized payments require authentication."
+                                                },
+                                                "allowInstallmentsMerge": {
+                                                    "type": "boolean",
+                                                    "description": "By default (when `false`), on a multi-seller purchase is on the run, a simple intersection with installments options configured by every seller will be available. When indicated as `true`, this option allows a more complex but flexible installment option, since it considers max installments of every seller configuration, even if those don't match. Installment values may not be equal in this case."
```

Comment:
```suggestion
                                                    "description": "By default (when `false`), on a multi-seller purchase is on the run, a simple intersection with installment options configured by every seller will be available. When indicated as `true`, this option allows a more complex but flexible installment option, since it considers max installments of every seller configuration, even if those do not match. Installment values may not be equal in this case."
```